File: examples/scala/src/main/scala/invalid/Sample.scala
package invalid

import org.slf4j.LoggerFactory

import java.nio.file.Paths
import java.lang.Boolean

object Sample {

    def main(args: Array[String]) = {
        // method added in 1.7
        Boolean.compare(true, true)
    }

    def someth() {
        // class added in 1.7
        Paths.get("/tmp")
        LoggerFactory.getLogger("sample logger")
    }
}

File: examples/java/src/main/java/invalid/Sample.java
package invalid;

import javax.naming.InitialContext;

public class Sample {

    public static void main(String[] args) {
        // method added in 1.7
        Boolean.compare(true, true);
    }

    public void someth() throws Exception {
        // not available in android
        InitialContext.doLookup("some");
    }
}